static/src/javascripts/projects/common/modules/commercial/hosted-gallery.js

```diff
@@ -275,7 +277,7 @@ define([
                 if(this.useSwipe){
                     this.translateContent(this.index, 0, 100);
                 }
-
+                omniture.trackLinkImmediate(config.page.analyticsName + ' - image ' + this.index);
```

Comment:
This is called when changing between images (can be by scrolling, swiping or arrow buttons) which is all controlled by the javascript. So, nowhere to really put that attribute. Maybe trackLinkImmediate isn't the right omniture method to call? I'm not familiar at all with omniture! What do you think is best?